src/mon/OSDMonitor.cc

```diff
@@ -10178,6 +10178,12 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       goto reply;
     }
     if (rel == CEPH_RELEASE_MIMIC) {
+      if (!mon->monmap->get_required_features().contains_all(
+	    ceph::features::mon::FEATURE_MIMIC)) {
```

Comment:
why is this needed? hope it's not too important since existing mimic won't have it